src/main/webapp/app/forms/form-footer/form-footer.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="update-programming-exercise-footer gap-4">
+    <jhi-exercise-update-notification
+        class="flex-grow-1"
+        [isCreation]="isCreation() ?? false"
+        [isImport]="isImport() ?? false"
```

Comment:
The artemis docs about that have changed, since signal calls are the one exception for template function calls, as they have built in change detection and therefore prevent performance issues.

https://docs.artemis.cit.tum.de/dev/guidelines/client.html
https://angular.io/guide/signals

Reply:
Makes sense as this is then not a function call but related to the signals. An important difference!